backend/tests/Tests/SqlCompiler.Tests.fs

```diff
@@ -256,4 +303,4 @@ let partialEvaluation =
 let tests =
   testList
     "SqlCompiler"
-    [ inlineWorksAtRoot; inlineWorksWithNested; partialEvaluation; compileTests ]
+    [ inlineWorksAtRoot; inlineWorksWithNested; inlineWorksWithPackageFunctionsSqlBinOp; inlineWorksWithPackageFunctionsSqlFunction; partialEvaluation; compileTests ]
```

Comment:
These are a good start. Can you add more complex tests here, maybe the same complex ones above (using anotherNestedUserFn and `let a = 1 in let b = 9 in let c = useAdd 6 4 in (p.height == c) && (p.age == b) )`)